service/warehouse/db/query.sql

```diff
@@ -7,3 +7,12 @@ DO UPDATE SET
     branch_id = EXCLUDED.branch_id,
     updated_at = NOW()
 RETURNING *;
+
+-- name: GetRacks :many
+SELECT a.id, a.branch_id, a.name
+FROM warehouse.racks a
+LEFT JOIN warehouse.warehouse_racks b ON a.id = b.rack_id
+WHERE a.branch_id = $1 
+    AND a.name LIKE $2 
+    AND CASE WHEN
+        @is_get_all::bool THEN TRUE ELSE b.rack_id IS NULL END;
```

Comment:
`@is_get_all::bool THEN TRUE ELSE b.rack_id IS NULL END;`

jadi

`@is_get_available::bool THEN b.rack_id IS NULL ELSE TRUE END;`